game/fighter.py

```diff
@@ -84,15 +93,17 @@ def resource_path(relative_path):
             base_path = os.path.abspath(".")
 
         return os.path.join(base_path, relative_path)
-
+    
     def move(self, screen_width, screen_height, surface, target, obstacles):
-        GRAVITY = 1
+        GRAVITY = 2
         self.dx = 0
         self.dy = 0
 
         # check player 1 movement
-
-        self.keybinds(self.controls, surface, target,None)
+        if self.ai:  # if ai instance call fighter ai func
+            self.keyAi( surface, target,None)
+        else:
+            self.keybinds(self.controls, surface, target,None)
```

Comment:
same here you should add all this to your ai fighter class in some move_ai function

Reply:
i fixed this problem created a move function in fighter_ai